CMakeLists.txt

```diff
@@ -193,42 +193,40 @@ if(UMF_BUILD_LIBUMF_POOL_JEMALLOC AND (LINUX OR WINDOWS))
     )
 endif()
 
-# set UMF_PROXY_LIB_ENABLED
-if(LINUX OR WINDOWS)
-    # TODO: enable the proxy library in the Debug build on Windows
-    if(WINDOWS AND NOT (CMAKE_BUILD_TYPE STREQUAL "Release"))
+# set UMF_PROXY_LIB_ENABLED TODO: enable the proxy library in the Debug build on
```

Comment:
please add an empty line (like `#  `) between the actual comment and the TODO for the TODO to be in a separate line

Reply:
Done